### ISSUE TITLE
Release 0.3.1

### ISSUE THREAD

**Issue by ericsagnes** [styx-static/styx#20]
Release a version with the bug fixes for #18 and #19.

@danbst Is there anything else you would like to see in this release?

**Comment by danbst** [comment]
I haven't extensively tested yet. I'm ok with using `latest.tar.gz`, I don't plan to use releases =)

**Comment by danbst** [comment]
No, I have one more request. Previously there was at least one post in new generated site. Now there isn't. I think this is related to problem I have:

```
$ styx preview
these derivations will be built:
  /nix/store/zdv0y0fdvjw0y8xgbbszck3ahff3ln5h-styx-site.drv
building path(s) ‘/nix/store/a14004z93l3dhf88xb823fbsnfdxirrp-styx-site’
server listening on http://127.0.0.1:8080
press Ctrl+C to stop

$ curl 127.0.0.1:8080
404 Not Found
$ ls /nix/store/a14004z93l3dhf88xb823fbsnfdxirrp-styx-site
$
$ ls -lah
total 28K
drwxr-xr-x 4 danbst users 4.0K Oct 29 04:17 .
drwxr-xr-x 3 danbst users 4.0K Oct 29 04:23 ..
drwxr-xr-x 2 danbst users 4.0K Oct 29 04:17 data
drwxr-xr-x 2 danbst users 4.0K Oct 29 04:17 themes
-rw-r--r-- 1 danbst users  251 Oct 29 04:17 conf.nix
-rw-r--r-- 1 danbst users 1.7K Oct 29 04:17 readme.md
-rw-r--r-- 1 danbst users 2.8K Oct 29 04:17 site.nix
```

**Comment by ericsagnes** [comment]
Yes since 0.3.0, there is no default content anymore as everything moved to themes and their example sites.

On the other hand, in `readme.md`, you can find instructions to install the showcase theme and preview its example site.

```
git submodule add https://github.com/styx-static/styx-theme-showcase.git themes/showcase
styx preview --in themes/showcase/example
```

`readme.md` also give a simple introduction on how to create a hello world page, this should be improved and more detailed though.

**Comment by danbst** [comment]
yeah, I didn't even consider reading readme.md =) ok so then

**Comment by ericsagnes** [comment]
All good then!
Released 0.3.1 in https://github.com/styx-static/styx/commit/19d73974e289f1b3c32f99f0f8da66ab276a5be5.
